package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.4.3",
+  "version": "0.4.4",
```

Comment:
Can we bump to `0.5` instead? Just because it's a pretty "big" change we're doing